jscomp/syntax/src/res_printer.ml

```diff
@@ -4730,6 +4730,20 @@ and print_arguments ~state ?(partial = false)
     in
     Doc.concat [Doc.lparen; arg_doc; Doc.rparen]
   | args ->
+    (* Avoid printing trailing comma when there is ... in function application *)
+    let hasDotDotDot, printed_args =
```

Comment:
would you do a little search if this functionality already exists somewhere else?
I kind of don't even remember using `...` verbatim as label, instead of some other attribute.

Reply:
This is what I see in the parser `          let attrs = if is_partial then [res_partial_attr] else [] in
`
so I'm not sure your code runs? In any case, adding a test will answer this.

Reply:
I wonder which is better, pattern match on the label or the expression like so, 

https://github.com/shulhi/rescript-compiler/blob/8a28dafe71082da9d40eb53ea182dc2eb85fc5f8/jscomp/syntax/src/res_parsetree_viewer.ml#L51-L58

Reply:
The label does not exist. In the ast.

Reply:
Think of it as app1(f, args) vs app2(f, args)

One being total and the other partial.
Internally the distinction is only the presence of the attribute.

Reply:
So "..." only exists as string in the source, but that's not how it's represented in the AST.